README.md

```diff
@@ -1,26 +1,29 @@
 # Sandman
 
-Sandman is a device that is intended to assist people, particularly those with disabilities, in using hospital style beds. Therefore, it is not just software, but a combination of both software and hardware. The software is primarily designed to enable controlling a bed by voice. However, work is also underway on providing analytics of usage through a web interface. Currently the software is intended to be run on a Raspberry Pi, and is primarily developed on a Raspberry Pi 4. 
+----> what beds are supported?  Is this common or specific to your bed manufacturer?
```

Comment:
The current method of controlling the bed will work with any motorized bed that uses a hand control.